CONTRIBUTORS.md

```diff
@@ -199,6 +199,7 @@ Created as part of the release process.
 + Lei Wang
 + Leo Kim
 + Lex Spoon
++ Liam Deacon
```

Comment:
Why is this updated?

Reply:
Using these steps: https://www.pantsbuild.org/2.18/docs/contributions/releases/release-process#bump-the-version
Ran this command: ./pants run src/python/pants_release/start_release.py -- --new 2.21.0.dev7 --release-manager ravi2024sk --publish
This command updated version .md  and contributors.md file. If this is not requiored I can revert this change. Sorry for the confusion.